lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -26,18 +26,26 @@ module.exports = function(SupportedFormatsModel, SelectionStateModel, AnalyticsS
    * @type {String[]}
    * @public
    */
-  this.mainSupportedExtensions = _.slice(SupportedFormatsModel.getAllExtensions(), 0, 3);
+  this.mainSupportedExtensions = _.intersection([
```

Comment:
Oh, never mind; I read this as "union" instead of "intersection". That makes more sense. Although, to be honest, if this code is ever necessary I think Etcher has a serious problem :laughing: